main.js

```diff
@@ -215,10 +215,10 @@ $catalog.addEventListener('click', function (event) {
 
 $details.addEventListener('click', function (event) {
   var $clicked = event.target
-  if ($clicked.closest('.btn-primary') !== null) {
+  if ($clicked.closest('#add') !== null) {
```

Comment:
We're not really searching for the "closest" button, we are checking if this is "the" button.